cdap-watchdog/src/main/java/co/cask/cdap/logging/guice/LoggingModules.java

```diff
@@ -45,6 +47,12 @@
  */
 public class LoggingModules extends RuntimeModule {
 
+  private final CConfiguration cConf;
+
+  public LoggingModules(CConfiguration cConf) {
```

Comment:
Shouldn't take a `CConfiguration` in constructor, but it should be acquired through Guice Injector

1. It involves a lot of changes
2. There can be configs set/add through the ConfigModule. Pass it directly will lack those changes.

The change would be simpler if you just have the corresponding reader (e.g. DistributedLogReader, KafkaLogReader) to take a `CConfiguration` in the constructor as those constructors are already annotated with `@Inject`.